test/supertestSetup.js

```diff
@@ -109,8 +109,10 @@ const appSetup = (route, user = 'caUser', prefix = '', flash = jest.fn().mockRet
     res.locals.user = userObj
     next()
   })
-  app.use(passport.initialize())
+
   app.use(cookieSession({ keys: [''] }))
+  app.use(passport.initialize())
```

Comment:
passport requires both these or get `req.isAuthenticated() not a function` type error